README.md

```diff
@@ -1 +1,12 @@
 # retail-url-scraper
+A web scraper that scrapes retail websites and returns URLs for all listed products. Runs on Express and Node
+
+To Install and run:
+
+1. npm install
+
+2. DEBUG=site-urls:* npm start
```

Comment:
I think DEBUG env should be part of package.json scripts and here leave just:
`npm start`